src/axom/primal/tests/primal_winding_number.cpp

```diff
@@ -116,6 +116,58 @@ TEST(primal_winding_number, simple_cases)
               abs_tol);
 }
 
+TEST(primal_winding_number, rational_cases)
```

Comment:
It would be cool to also add a test with several rational quadratics or cubics defining a circle and compare the winding number of those against point containment in a `primal::Circle` of the same radius and center for a variety of test points